Content.Server/Storage/EntitySystems/StorageSystem.Fill.cs

```diff
@@ -39,8 +51,33 @@ private void OnStorageFillMapInit(EntityUid uid, StorageFillComponent component,
                 Logger.ErrorS("storage", $"Tried to StorageFill {entry.PrototypeId} inside {uid} but can't.");
                 EntityManager.DeleteEntity(ent);
             }
+        }
+
+        // handle orgroup spawns
+        foreach (var group in orGroupedSpawns)
+        {
+            Random r = new Random();
+            double diceRoll = r.NextDouble();
+            List<EntitySpawnEntry> shuffled = group.Value.OrderBy(a => r.Next()).ToList();
```

Comment:
Use the robustrandom dependency instead please.

Reply:
applied this